pkg/getty/README.md

```diff
@@ -10,7 +10,7 @@ If you want a console getty, add the following to your `moby.yml`:
 ```
 services:
   - name: getty
-    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
```

Comment:
@rn thanks for explaining.

@justincormack got it on the `Makefile`. But how do you structure that so `Makefile` is in but `README` (and other irrelevant stuff) is out?